doc/source/api/genotypephenotype.rst

```diff
@@ -0,0 +1,528 @@
+   :alt: image
+
+
+-  Coupled with sequence annotation and GA4GH datasets. Clients will
+   want implementation specific featureId/genotypeId to match and
+   integrate with the rest of the APIs.
+
+.. figure:: https://cloud.githubusercontent.com/assets/47808/14397306/7ad8bb70-fd91-11e5-9295-85c7034ce544.png
+   :alt: image
+
+
+-  Operating in concert with other instances of g2p servers where the
+   client's loosely federated query is supported by heterogeneous
+   server. Challenges: Normalizing API behavior across implementations
+   (featureId for given region different per implementation)
+
+.. figure:: https://cloud.githubusercontent.com/assets/47808/14397316/8a268b8e-fd91-11e5-907d-441fca3450cb.png
+   :alt: image
+
+
+Approach
+--------
+
+We based our original work on the model captured in `ga4gh/schemas
+commit of Jul 30,
+2015 <https://github.com/ga4gh/schemas/tree/be171b00a5f164836dfd40ea5ae75ea56924d316>`__.
+This version of the schema predates the `separated genotype to phenotype
+files from
+baseline <https://github.com/ga4gh/schemas/commit/846b711fdcf544bf889cc7dbab19c6c48e9a9428>`__.
+After on review of the schemas and code, the team had feedback about
+separation of responsibility in the original API. The API was refactored
+to separate the searches for genotype, phenotype, feature and
+associations.
+
+Data Model
+----------
+
+The cancer genome database `Clinical Genomics Knowledge
+Base <http://nif-crawler.neuinfo.org/monarch/ttl/cgd.ttl>`__ published
+by the Monarch project was the source of Evidence.
+
+.. figure:: https://cloud.githubusercontent.com/assets/47808/9338065/a0a84b8e-4597-11e5-82ed-65d7b9f3ae97.png
+   :alt: image
+
+
+Intent: The GA4GH Ontology schema provides structures for unambiguous
+references to ontological concepts and/or controlled vocabularies
+within Protocol Buffers. The structures provided are not intended for
+de novo modeling of ontologies, or representing complete ontologies
+within Protocol Buffers. References to e.g. classes from external
+ontologies or controlled vocabularies should be interpreted only in
+their original context i.e. the source ontology.
+
+Due to the flexibility of the data model, users have a number of options
+for specifying each query term
+`feature <https://github.com/ga4gh/schemas/blob/be171b00a5f164836dfd40ea5ae75ea56924d316/src/main/resources/avro/genotypephenotypemethods.avdl#L105>`__,
+`phenotype <https://github.com/ga4gh/schemas/blob/be171b00a5f164836dfd40ea5ae75ea56924d316/src/main/resources/avro/genotypephenotypemethods.avdl#L108>`__,
+and
+`evidence <https://github.com/ga4gh/schemas/blob/be171b00a5f164836dfd40ea5ae75ea56924d316/src/main/resources/avro/genotypephenotypemethods.avdl#L111>`__.
+
+API
+---
+
+The G2P schemas define several endpoints broken into two entity searches
+and an association search.
+
+A feature or phenotype can potentially be represented in increasing
+specificity as either [a string, an ontology identifier, an external
```

Comment:
Same as above. I believe the offering does not provide the entity or external identifier functionality.